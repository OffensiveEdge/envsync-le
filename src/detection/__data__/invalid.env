# Invalid .env file with syntax errors
DATABASE_URL=postgresql://user:pass@localhost:5432/db
API_KEY=sk-12345
DEBUG=true
PORT=3000

# Missing equals sign
INVALID_LINE

# Empty key
=VALUE_WITHOUT_KEY

# Malformed line
KEY WITH SPACES=value

# Comment line (should be ignored)
# This is a comment

# Empty line

NORMAL_KEY=normal_value
