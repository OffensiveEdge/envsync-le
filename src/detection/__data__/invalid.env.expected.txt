DATABASE_URL
API_KEY
DEBUG
PORT
NORMAL_KEY
